tokio/src/sync/mutex.rs

```diff
@@ -3,6 +3,8 @@ use crate::sync::batch_semaphore as semaphore;
 use std::cell::UnsafeCell;
 use std::error::Error;
 use std::fmt;
+use std::marker;
+use std::mem;
```

Comment:
I think that when importing a module from `std` (rather than names in a module), we tend to prefer
```suggestion
use std::{fmt, marker, mem};
```
but this is not a blocker.